client/src/js/services/FilterService.js

```diff
@@ -150,8 +150,8 @@ function FilterService(Store) {
 
     Object.keys(this._filterIndex).forEach(function (key) {
       var filter = this._filterIndex[key];
-
-      if (filter._value) {
+      
+      if (filter._value !== null && filter._value !== undefined && filter._value.length !== '') {
```

Comment:
I think you can just do `if (angular.isDefined(filter._value)) {`.

Reply:
```angular.isDefined``` will condider ```null``` and empty string as value and 
![filtres](https://user-images.githubusercontent.com/5418919/29930179-9631458e-8e65-11e7-9197-80743e11a50f.PNG)

Reply:
Fascinating.  I wonder how a user is able to enter `null`...